server/bin/pbench-server-activate-start-httpd

```diff
@@ -45,4 +45,4 @@ ln -s \
         ${pbench_users_dir} \
         ${pbench_static_dir} \
         ${docroot} 2>/dev/null
-exit $?
+exit 0
```

Comment:
Why do we need to do this?

Reply:
Seems like we should be capturing the status of the systemctl commands and firewall-cmds as well, failing when any of them don't work, including the creation of the links.

Reply:
The `ln -s` fails if the links exist already, so this would fail every time after the first time.

I'll move the firewall stuff into the playbook, but the only httpd module I found was for enabling/disabling apache modules. Do you know of anything else?

I'll rework this.